opendm/config.py

```diff
@@ -9,7 +9,7 @@
 import sys
 
 # parse arguments
-processopts = ['dataset', 'split', 'merge', 'opensfm', 'mve', 'odm_filterpoints',
+processopts = ['dataset', 'split', 'merge', 'opensfm', 'openmvs', 'odm_filterpoints',
```

Comment:
Squee! Ahem. I mean, an astute choice.

Reply:
OpenMVS does make some crazy-good meshes, huh?
https://arc-team-open-research.blogspot.com/2016/12/comparing-7-photogrammetry-systems.html

Reply:
Currently it has the best F-score of all FOSS programs on the tanks and temples benchmark: https://www.tanksandtemples.org/leaderboard/?table_0-sort=my_avg_rank